clients/gtest/matmul_gtest.yaml

```diff
@@ -544,7 +543,7 @@ Tests:
   scaleA: [ 0, 1]
   scaleB: [ 0, 1]
   bias_vector: [0, 1]
-  bias_type: f32_r
+  bias_type: bf16_r
   unit_check: 1
```

Comment:
The test cases can test for bias type != To or Talpha. And this is also the default bias type for F8SS gemm